CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+**FUTURE BREAKING CHANGES:**
+
+* **Storage:** on or about 2021-01-15 we are planning to remove the
+  `bucket_policy_only` attribute from `storage::BucketIamConfiguration` struct.
+  This attribute represents the pre-GA name for "Uniform Bucket Level Access"
+  becoming GA, applications can use `uniform_bucket_level_access`.
```

Comment:
s/GA, applications can use/GA.  Applications can use/?

Perhaps "can now use"?

Reply:
`uniform_bucket_level_access` has been available for over a year, I am going with "Applications should use ... instead".